From 85e798e860f634897d4a3e2b8582009818c96dc2 Mon Sep 17 00:00:00 2001
From: Matti Kosola <matti.kosola@jolla.com>
Date: Tue, 8 Oct 2019 13:03:24 +0300
Subject: [PATCH] Revert "mke2fs.conf: enable metadata_csum by default"

This reverts commit d0b9e0a6aa7d6805338a43b4e372623352d8df09.

Ext4 metadata_csum config will cause mounting problems with older
kernel versions when ext4 filesystem is created on machine which
is running relative new kernel version (4.9, 5.0, etc).
---
 misc/mke2fs.conf.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/misc/mke2fs.conf.in b/misc/mke2fs.conf.in
index 00afd912..63d5c588 100644
--- a/misc/mke2fs.conf.in
+++ b/misc/mke2fs.conf.in
@@ -11,7 +11,7 @@
 		features = has_journal
 	}
 	ext4 = {
-		features = has_journal,extent,huge_file,flex_bg,metadata_csum,64bit,dir_nlink,extra_isize
+		features = has_journal,extent,huge_file,flex_bg,uninit_bg,64bit,dir_nlink,extra_isize
 		inode_size = 256
 	}
 	small = {
-- 
2.21.0

